services/src/api/model/operators.rs

```diff
@@ -75,13 +160,114 @@ pub struct VectorResultDescriptor {
     pub bbox: Option<BoundingBox2D>,
 }
 
+impl From<geoengine_operators::engine::VectorResultDescriptor> for VectorResultDescriptor {
+    fn from(value: geoengine_operators::engine::VectorResultDescriptor) -> Self {
+        Self {
+            data_type: value.data_type.into(),
+            spatial_reference: value.spatial_reference.into(),
+            columns: value
+                .columns
+                .into_iter()
+                .map(|(key, value)| (key, value.into()))
+                .collect(),
+            time: value.time.map(Into::into),
+            bbox: value.bbox.map(Into::into),
+        }
+    }
+}
+
+impl From<VectorResultDescriptor> for geoengine_operators::engine::VectorResultDescriptor {
+    fn from(value: VectorResultDescriptor) -> Self {
+        Self {
+            data_type: value.data_type.into(),
+            spatial_reference: value.spatial_reference.into(),
+            columns: value
+                .columns
+                .into_iter()
+                .map(|(key, value)| (key, value.into()))
+                .collect(),
+            time: value.time.map(Into::into),
+            bbox: value.bbox.map(Into::into),
+        }
+    }
+}
+
+impl ResultDescriptor for VectorResultDescriptor {
```

Comment:
do we acutally need/use the methods defined here?

Reply:
At some point during development of this PR these methods were called. I looked into it and this is no longer the case. [f79ac16](https://github.com/geo-engine/geoengine/pull/662/commits/f79ac16506c23a31c64317304f89efd420ce7900)